ios/ElectrodeApiImpl/APIImpls/Navigation/ENNavigationUtils.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ENNavigationUtils.swift
+//  ElectrodeApiImpl
+//
+//  Created by Jeffrey Wang on 4/15/21.
+//  Copyright © 2021 Walmart. All rights reserved.
+//
+
+import Foundation
+
+open class ENNavigationUtils {
+
+    open func emitPayloadToCurrentRNView(navigationController: UINavigationController, jsonPayload: String?) {
```

Comment:
Can we make this as a method inside the BaseVC and MiniAppVC itself? 

Assumption is, A VC should always be responsible for sending data to a view that it hosts. Making this as a util makes it a method that is decoupled from the current flow.

Reply:
Moved to base class, which then calls on miniappvc class